src/main/scala/com/gu/dynamodbswitches/Switches.scala

```diff
@@ -35,6 +34,27 @@ trait Switches extends Logging {
         error(s"Encountered Amazon service error when trying to update switches from DynamoDB: ${exception.getMessage}")
     }
   }
+
+    def updateDynamo(switches : List[Switch]) = {
+    val listWR = switches.map(switch => {
+      new WriteRequest(new PutRequest(Map("name" -> switch.toStringAttribute(), "enabled" -> switch.toNumAttributeValue()).asJava))
+    }).asJava
+
+    try {
+      dynamoDbClient.batchWriteItem(Map(dynamoDbTableName -> listWR).asJava)
+    } catch {
+      case e: Exception => println(e)
+      case _ => println("Something went wrong")
+    }
+  }
+
+  def getSwitchboardState(): List[Switch] = {
+    val results = dynamoDbClient.scan(new ScanRequest(dynamoDbTableName)).getItems.asScala.toList.map(_.asScala.toMap)
+    results.map(item => {
+      Switch(item("name").getS, item("enabled").getN == "1")
+    })
```

Comment:
No need for the inside `{}`
You also don't need the intermediary val `results` and the double `map`. What about:
```
dynamoDbClient
  .scan(new ScanRequest(dynamoDbTableName))
  .getItems.asScala.toList
  .map { 
    val item = _.asScala.toMap
    Switch(item("name").getS, item("enabled").getN == "1")
  }
```